src/Excel/ExportCoreReport.cs

```diff
@@ -399,9 +407,29 @@ private void Generate_All_VM_IaaS_Server_Rehost_Perf_Worksheet()
                 dataWs.Cell(2, 1).InsertData(All_VM_IaaS_Server_Rehost_Perf_List);
         }
 
+        private void Generate_AVS_Summary_Worksheet()
+        {
+            var dataWs = CoreWb.Worksheets.Add(CoreReportConstants.AVS_Summary_TabName, 16);
+
+            UtilityFunctions.AddColumnHeadersToWorksheet(dataWs, CoreReportConstants.AVS_Summary_Columns);
+
+            if (AVS_Summary_List != null && AVS_Summary_List.Count > 0)
+                dataWs.Cell(2, 1).InsertData(AVS_Summary_List);

```

Comment:
Need to add AVS AsOnPrem costs to AVS Summary Worksheet.

Reply:
have already been handled